### PR TITLE
docs(README): Add more usage info

### PR DESCRIPTION
**Pull request by korthout**
This update brings a more comprehensive and user friendly readme. In addition, it has more similarities to the README in the `stryker` repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
+If you plan on using Stryker in more projects, the Stryker-CLI is the easiest way to install, configure and run Stryker for your project.
 
-The Stryker CLI can be easily installed using NPM.
+# Installation
+The Stryker-CLI can be easily installed using NPM.
 
 ```
-> npm install -g stryker-cli
+$ npm install -g stryker-cli
 ```
```
Comment (nicojs) [inline README.md]: Maybe we can clarify this even more. Something like:

The Stryker-CLI installation does **NOT** install `stryker` itself. Instead, it only passes along commands to your local `stryker` (i.e. the stryker installation contained in your `node_modules` folder), or prompt to install a local stryker if none could be found. Therefore you should *always install it globally* with `-g` or `--global`. If you want to use stryker directly from [npm scripts](https://docs.npmjs.com/files/package.json#scripts), feel free to do so without stryker-cli.

**Comment by korthout** [comment]
Hi @nicojs I've updated the Readme a bit more, using what you said, but also expanding a bit more. How do you feel about this?

File: README.md [hunk 2]
```diff
@@ -24,10 +26,10 @@ For example, you can kick off mutation testing using:
 $ stryker run
 ```
 
-Please execute the Styker-CLI commands from your project's root directory.
+Please always execute the Styker-CLI commands from your project's root directory. Although we are able to find your local Stryker installation in parent folders, running Stryker from a different folder than your project's root directory may result in unexpected behavior.
```
Comment (nicojs) [inline README.md]: This is actually becausr of this bug: https://github.com/stryker-mutator/stryker/issues/245

Comment (korthout) [inline README.md]: Ah interesting, I'll probably take a look at this issue soon, at least keep watch on it. I will update the readme accordingly when it is solved.

**Pull request merged**